queries.sql

```diff
@@ -123,3 +123,65 @@ GROUP BY o.full_name
 ORDER BY COUNT(*) DESC
 LIMIT 1;
 
+SELECT a.name AS animal_name
+FROM animals AS a
+INNER JOIN visits AS v ON a.id = v.animal_id
+WHERE v.vet_id = (SELECT id FROM vets WHERE name = 'William Tatcher')
+ORDER BY v.visit_date DESC
+LIMIT 1;
+--
+SELECT COUNT(DISTINCT v.animal_id) AS animal_count
+FROM visits AS v
+WHERE v.vet_id = (SELECT id FROM vets WHERE name = 'Stephanie Mendez');
+--
+SELECT v.name AS vet_name, s.name AS specialty_name
+FROM vets AS v
+LEFT JOIN specializations AS sp ON v.id = sp.vet_id
+LEFT JOIN species AS s ON sp.species_id = s.id;
+--
+SELECT a.name AS animal_name
+FROM animals AS a
+INNER JOIN visits AS v ON a.id = v.animal_id
+INNER JOIN vets AS ve ON v.vet_id = ve.id
+WHERE ve.name = 'Stephanie Mendez'
+  AND v.visit_date >= DATE '2020-04-01'
+  AND v.visit_date <= DATE '2020-08-30';
+
+--
+SELECT a.name AS animal_name, COUNT(*) AS visit_count
+FROM animals AS a
+INNER JOIN visits AS v ON a.id = v.animal_id
+GROUP BY a.name
+ORDER BY visit_count DESC
+LIMIT 1;
+
```

Comment:
- [ ] @karanJ2212 great work on your queries, however since you forgot to add some data, this query result was incorrect, kindly consider updating the screenshot result

Reply:
updated